diff --git a/node_modules/gatsby-link/index.js b/node_modules/gatsby-link/index.js
index dcad82d..133440b 100644
--- a/node_modules/gatsby-link/index.js
+++ b/node_modules/gatsby-link/index.js
@@ -179,7 +179,6 @@ var GatsbyLink = /*#__PURE__*/function (_React$Component) {
           _onMouseEnter(e);
         }
 
-        ___loader.hovering((0, _parsePath.parsePath)(to).pathname);
       },
       onClick: function onClick(e) {
         if (_onClick) {
